runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -940,19 +940,22 @@
 	BufferProvider getBufferProvider();
 
 	/**
-	 * Gets the amount of ticks since the last mouse movement occurred.
+	 * Gets the amount of client ticks (see CLIENT_TICK_LENGTH in {@link Constants})
+	 * since the last mouse movement occurred.
 	 *
 	 * @return amount of idle mouse ticks
 	 */
 	int getMouseIdleTicks();
 
 	/**
-	 * Gets the number of milliseconds since the last mouse press occurred.
+	 * Gets the time at which the last mouse press occurred in milliseconds since
+	 * 1st January 1970 00:00
```

Comment:
I'd be fine with simply referring to this as the UNIX epoch, since that's fairly well-understood by people who deal with time in that format.